app/main.py

```diff
@@ -24,6 +26,20 @@
 )
 
 
+@app.middleware("http")
+async def log_requests(request: Request, call_next):
```

Comment:
I think it's important to attach the id to the request somehow. See [this](https://medium.com/gradiant-talks/identifying-fastapi-requests-in-logs-bac3284a6aa) and [this](https://medium.com/@sondrelg_12432/setting-up-request-id-logging-for-your-fastapi-application-4dc190aac0ea)

(btw, I'm ok with using the integrated logging package, if you think we should use another one [as loguru] please exlpain why)

Reply:
I understand that Loguru already has context variable management, however I also prefer to continue using the Logging library. It was necessary to create the context variable in a separate file to be able to call it at the different points where the logs can be implemented.